### ISSUE TITLE
KeyboardAvoidingView: prefersCrossFadeTransitions is undefined

### ISSUE THREAD

**Issue by henrik-d** [facebook/react-native#34784]
### Description

Some devices on iOS report the following error:

```
TypeError undefined is not a function 
    node_modules/react-native/Libraries/Components/Keyboard/KeyboardAvoidingView.js:88:31
```

It seems to be an issue with `AccessibilityInfo.prefersCrossFadeTransitions()`.

### Version

0.70.1

### Output of `npx react-native info`

```
System:
    OS: macOS 12.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 2.08 GB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.6 - ~/.nvm/versions/node/v14.17.6/bin/node
    Yarn: 1.22.19 - ~/.yarn/bin/yarn
    npm: 8.3.2 - ~/.nvm/versions/node/v14.17.6/bin/npm
    Watchman: 4.7.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /Users/henrik/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 25, 29, 30, 31
      Build Tools: 27.0.2, 27.0.3, 28.0.2, 28.0.3, 29.0.0, 29.0.2, 30.0.2, 30.0.3, 31.0.0, 32.0.0, 32.1.0, 33.0.0, 33.0.0
      System Images: android-19 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom, android-30 | Google APIs Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.14.1 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0 
    react-native: 0.70.1 => 0.70.1 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

```

### Steps to reproduce

I don't know, happens only on some devices.

### Snack, code example, screenshot, or link to a repository

-

**Comment by ildfreelancer** [comment]
@henrik-d 
because in latest source, they didn't have it
https://github.com/facebook/react-native/blob/0.70-stable/Libraries/Components/AccessibilityInfo/AccessibilityInfo.js

I got the crash too.

**Comment by gabrieldonadel** [comment]
That's because of https://github.com/reactwg/react-native-releases/discussions/34#discussioncomment-3656859

**Comment by fungilation** [comment]
Reproducing same. On iOS 15.7, iPhone 12

**Comment by cortinico** [comment]
> That's because of [reactwg/react-native-releases#34 (comment)](https://github.com/reactwg/react-native-releases/discussions/34#discussioncomment-3656859)

Thanks for mentioning this @gabrieldonadel 
Seems like the issue is effectively already solved on `main` and we need to release a new point release that include the fix (it will happen in the immediate future).

**Comment by svenlombaert** [comment]
Seems like this is now released in 0.70.2:
https://github.com/facebook/react-native/releases/tag/v0.70.2
